pkg/asset/installconfig/aws/platform.go

```diff
@@ -3,38 +3,18 @@ package aws
 import (
 	"fmt"
 	"sort"
-	"strings"
 
 	survey "github.com/AlecAivazis/survey/v2"
-	"github.com/AlecAivazis/survey/v2/core"
 	"github.com/sirupsen/logrus"
 
 	"github.com/openshift/installer/pkg/types/aws"
-	"github.com/openshift/installer/pkg/version"
 )
 
 // Platform collects AWS-specific configuration.
 func Platform() (*aws.Platform, error) {
-	architecture := version.DefaultArch()
-	regions := knownPublicRegions(architecture)
-	longRegions := make([]string, 0, len(regions))
-	shortRegions := make([]string, 0, len(regions))
-	for id, location := range regions {
-		longRegions = append(longRegions, fmt.Sprintf("%s (%s)", id, location))
-		shortRegions = append(shortRegions, id)
-	}
-	var regionTransform survey.Transformer = func(ans interface{}) interface{} {
-		switch v := ans.(type) {
-		case core.OptionAnswer:
-			return core.OptionAnswer{Value: strings.SplitN(v.Value, " ", 2)[0], Index: v.Index}
-		case string:
-			return strings.SplitN(v, " ", 2)[0]
-		}
-		return ""
-	}
-
+	regions := knownPublicRegions()
 	defaultRegion := "us-east-1"
-	if !IsKnownPublicRegion(defaultRegion, architecture) {
```

Comment:
I don't think I understand why architecture is removed as a filter for known public regions.